src/scripts/blocks/countdown-timer/DisplayComponent.js

```diff
@@ -0,0 +1,177 @@
+const { applyFilters } = wp.hooks;
+const { PostMediaSelector } = benenson.components;
+
+class DisplayComponent extends Component {
+  componentDidMount() {
+    const { attributes, setAttributes } = this.props;
+
+    if (!attributes.countdownId) {
+      setAttributes({
+        countdownId: randId(),
+      });
+    }
+  }
+
+  render() {
+    const { attributes, setAttributes } = this.props;
+    const {
+      countdownId = '',
+      date = new Date(),
+      title = '',
+      revealTitle = '',
+      revealContent = '',
+      revealBtnText = '',
+      revealBtnUrl = '',
+      backgroundColor = '',
+      backgroundId = null,
+      backgroundUrl = '',
+      textColor = '',
+      hideTimer = false,
+    } = attributes;
+
+    // Note: US English spelling.
+    const colourOptions = applyFilters('benenson.block.blockCountdown.colorOptions', [{
+      label: __('White', 'benenson'),
+      value: '',
+    }, {
+      label: __('Black', 'benenson'),
+      value: 'black',
+    }]);
+
+    const styles = {
+      backgroundImage: `url('${backgroundUrl}')`,
+      backgroundColor,
+    };
+
+    const classes = classnames('countdownTimer', {
+      [`is-${textColor}`]: !!textColor,
+    });
+
+    return (
+      <Fragment>
+        <InspectorControls>
+          <PanelBody title={ __('Options', 'benenson') }>
+            <ToggleControl
+              label={ __('Hide timer', 'benenson') }
+              help={ __('When the timer has finished hide it from users.', 'benenson') }
+              checked={ hideTimer }
+              onChange={ hideShow => setAttributes({ hideTimer: hideShow }) }
+            />
+            <SelectControl
+              label={ __('Text Colour', 'benenson') }
+              value={ textColor }
+              onChange={ color => setAttributes({ textColor: color }) }
+              options={ colourOptions }
+            />
+            <DateTimePicker
+              currentDate={ date }
+              onChange={ newDate => setAttributes({ date: newDate }) }
+            />
+          </PanelBody>
+          <PanelBody title={ __('Background colour', 'benenson') }>
+            <ColorPicker
+              color={ backgroundColor }
+              onChangeComplete={ color => setAttributes({ backgroundColor: color.hex }) }
+              disableAlpha={ true }
+            />
+            <Button
+              className="components-button is-button is-default is-large"
+              onClick={ () => setAttributes({ backgroundColor: '' }) }
+            >
+            { __('Remove background colour', 'benenson') }
+            </Button>
+          </PanelBody>
+          <PanelBody title={ __('Background Image', 'benenson') }>
+            <PostMediaSelector
+              onUpdate={ (media) => {
+                  setAttributes({
+                    backgroundUrl: media ? media.source_url : '',
+                    backgroundId: media ? media.id : null,
+                  });
+                }
+              }
```

Comment:
Please place these curly braces on one line (spaced apart), and reduce indentation of function body by one level.